examples/main.go

```diff
@@ -35,7 +44,18 @@ func main() {
 			return
 		}
 		// Log raw message struct
-		fmt.Println(mes)
+		//fmt.Println(mes)
```

Comment:
Yeah maybe.. Thinking about it I'm actually not sure how to remove WaitFor() (and then also removing the channel). But I have made the channel private, and then we can make Start() blocking until the bot is finished.